src/cli/main.cpp

```diff
@@ -37,8 +37,8 @@ std::string main_help()
 int main(int argc, char* argv[])
    {
    std::cerr << Botan::runtime_version_check(BOTAN_VERSION_MAJOR,
-                                             BOTAN_VERSION_MINOR,
-                                             BOTAN_VERSION_PATCH);
+             BOTAN_VERSION_MINOR,
+             BOTAN_VERSION_PATCH);
 
```

Comment:
Does this change make sense? I am used to the following two indentation methods from other technologies (Python, Swift):

1. break all arguments and use default indent level

```
std::cerr << Botan::runtime_version_check(
   BOTAN_VERSION_MAJOR,
   BOTAN_VERSION_MINOR,
   BOTAN_VERSION_PATCH
);
```

2. Indent to first argument start

```
std::cerr << Botan::runtime_version_check(BOTAN_VERSION_MAJOR,
                                          BOTAN_VERSION_MINOR,
                                          BOTAN_VERSION_PATCH);
```

So the original code looks good to me the change does not improve things in case rule 1. and 2. are desired.